cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -225,10 +225,10 @@ public static void closeAdminClientWithTimeout(AdminClient adminClient, long tim
       String securityProtocol = configs.getString(AdminClientConfig.SECURITY_PROTOCOL_CONFIG);
       adminClientConfigs.put(AdminClientConfig.SECURITY_PROTOCOL_CONFIG, securityProtocol);
       setStringConfigIfExists(configs, adminClientConfigs, SaslConfigs.SASL_MECHANISM);
-      setStringConfigIfExists(configs, adminClientConfigs, SaslConfigs.SASL_JAAS_CONFIG);
+      setPasswordConfigIfExists(configs, adminClientConfigs, SaslConfigs.SASL_JAAS_CONFIG);
 
       // Configure SSL configs (if security protocol is SSL)
-      if (securityProtocol.equals(SecurityProtocol.SSL.name)) {
+      if (securityProtocol.equals(SecurityProtocol.SSL.name) || securityProtocol.equals(SecurityProtocol.SASL_SSL.name)) {
```

Comment:
Can we 
(1) make the same changes in `CruiseControlMetricsUtils#addSslConfigs(Properties, CruiseControlMetricsReporterConfig)`?
(2) update the comment above from `Configure SSL configs (if security protocol is SSL)` to `Configure SSL configs (if security protocol is SSL or SASL_SSL)`?

Reply:
I pushed the requested changes